services/elasticache/pkg/resource/replication_group/custom_update_api_test.go

```diff
@@ -410,7 +410,8 @@ func TestNewIncreaseReplicaCountRequestPayload(t *testing.T) {
 	// Tests
 	t.Run("EmptyPayload=NoSpec", func(t *testing.T) {
 		desired := provideResource()
-		payload, err := rm.newIncreaseReplicaCountRequestPayload(desired)
+		latest := provideResource()
```

Comment:
I was initially a little confused by this because the name of the function `provideResource` seems to indicate that some filled-in `resource` is returned, but actually it's an *empty* `resource`.

So, might be good to rename the `providerResource` function to `emptyResource` or similar :)